Adding Assistant space (#2075)
huggingface/blog: README.md

## How to Use Highlight Boxes

See an example [in this PR](https://github.com/huggingface/blog/pull/1180). This is to be used sparingly :)

## More Assistance

Feel free to check out this [assistant space](https://huggingface.co/spaces/derek-thomas/hugging-face-blog-assistant) 
to help with rending some of the tricky markdown like:
- Inserting caption figures with image tables
- Tip formatting
- Inserting clickable references
 
Do Give it a like if it was helpful.

> Also you can use the [community blog editor](https://huggingface.co/new-blog) which has a preview function.

Reviewer: maybe invert the order and put this one first

Author: Done

## How to Use Highlight Boxes

See an example [in this PR](https://github.com/huggingface/blog/pull/1180). This is to be used sparingly :)

## More Assistance

You can use the [community blog editor](https://huggingface.co/new-blog) which has a preview function.

> Feel free to check out this [assistant space](https://huggingface.co/spaces/derek-thomas/hugging-face-blog-assistant) 

Reviewer 2: ```suggestion
Or check out this [assistant space](https://huggingface.co/spaces/derek-thomas/hugging-face-blog-assistant) 
```

> to help with rending some of the tricky markdown like:

Reviewer 2: ```suggestion
to help with rendering some tricky markdown blocks like:
```
